src/App.js

```diff
@@ -1,25 +1,54 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import Movies from './components/Movies'
+import movieData from './moviesData'
+import IndividualMovie from './components/IndividualMovie'
+import './styles/App.scss';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      movies: movieData,
+      isSingleMovie: false,
+      movie: ''
+    }
+  }
+
+  displayIndividualMovie = id => {
+    if (id) {
+      this.setState({
+        isSingleMovie: true,
+        movie: {
+          id: 1,
+          title: "Fake Movie Title",
+          poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg",
+          backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg",
+          release_date: "2019-12-04",
+          overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!",
+          average_rating: 6,
+          genres: ["Drama", "other things"],
+          budget:63000000,
+          revenue:100853753,
+          runtime:139,
+          tagline: "It's a movie!"
+        }
+      })
+    }
+  }
+
+  displayHomePage = () => {
+    this.setState({ isSingleMovie: false })
+  }
+
+  render() {
+    return (
+      <main className="main-container">
+        <h1>Rotten Tomatillos</h1>
+        <h2 className="opening-line">Where your imagination comes to life on the big screen</h2>
+        {!this.state.isSingleMovie ? <Movies movies={this.state.movies} findMovie={this.displayIndividualMovie} /> : <IndividualMovie movie={this.state.movie} displayHomePage={this.displayHomePage}/>}
```

Comment:
Nice use of a ternary and getting the conditional rendering to work!  Learned a lot with using a boolean value to determine what view to show.